### PR TITLE
feature/functional library app

### PR DESCRIPTION
**Pull request by igakigongo**
Should allow the user to perform the following action
- Add a book to their catalog
- Edit the book
- Remove a book
- View all books in their catalog

Specific Requirements
- Use an in-memory datastore

Extra Tasks (Optional)
- Migrate the datastore from an In-Memory type to use Local Storage
- Migrate from Local Storage to Firebase or use both, but allow synchronization of the two

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
```
Comment (bellom) [inline index.html:3]: Kindly correct the indentation in this file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300&display=swap">
+		<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+		<link rel="stylesheet" href="./css/app.css">
+		<title>The Library of Dracula</title>
+	</head>
+	<body>
+		<main>
+			<div class="container mt-5" id="root">
+        <h1 class="text-center py-5">Personal Catalogue</h1>
+        <div class="col-12 py-3 section-add-book" id="section-add-book">
```
Comment (bellom) [inline index.html:22]: We are missing just one feature in this project 😞.
image
https://www.theodinproject.com/courses/javascript/lessons/library

As soon as we open the web app, we shouldn't be seeing the form to add a new button right.
I guess you should have a `new book' button which the user can toggle on to display `form` for adding a new book.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @igakigongo  :clap: 

### Awesome Design!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
Kindly check out my comments below

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: .eslintrc.json [hunk 3]
```diff
@@ -1,33 +1,28 @@
...
-        "quotes": [
-            "error",
-            "double"
-        ],
-        "semi": [
-            "error",
-            "always"
-        ]
-    }
+  "env": {
```
Comment (bellom) [inline .eslintrc.json:3]: @igakigongo, one more thing!
You made some changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/javascript)
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

Comment (igakigongo) [reply .eslintrc.json:3]: Updated the file, but I still don't see much of a value for linting in some of these early projects - especially now that some files use prototypes - so I have to go into each use case (function definition on the prototype) and keep disabling the specific warning.

I also can't add the necessary documentation I want. The comments for disabling the linter warnings are getting in the way. 

Anyway I hope it gets better.

**Review by Oghenebrume50 — APPROVED** [review]
### STATUS: APPROVED 

Hi @igakigongo this looks really good all changes have been made, and it works perfectly :+1:  please merge and move forward :slightly_smiling_face:  **This is approved** :+1: 

**RAPHAEL NORIODE** :sunglasses:

**Comment by igakigongo** [comment]
Dank je wel Heer Noriode Raphael.

**Pull request merged**